src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -2027,10 +2027,10 @@ Made with 💗 by Microsoft and the PowerToys community.</value>
     <value>Getting started</value>
   </data>
   <data name="Oobe_Launch.Text" xml:space="preserve">
-    <value>Launch</value>
+    <value>Open</value>
```

Comment:
where is this used

Reply:
I think this is a dead string
https://github.com/search?q=repo%3Amicrosoft%2FPowerToys+Oobe_Launch&type=code

Reply:
You are correct. But that's off-topic. Niels has already called out to look for dead strings in #32479.